quaint/src/connector/mysql/native/mod.rs

```diff
@@ -68,6 +68,8 @@ impl MysqlUrl {
     }
 }
 
+const SYSTEM_NAME: &'static str = "mysql";
```

Comment:
I also find the name a bit unintuitive, maybe `DB_SYSTEM_NAME` (here and everywhere)?